blocks/better_send_message_with_text[x20].js

```diff
@@ -0,0 +1,234 @@
+        }
+    ],
+
+    outputs: [
+        {
+            "id": "action",
+            "name": "Action",
+            "description": "Type: Action\n\nDescription: Executes the following blocks when this block finishes its task.",
+            "types": ["action"]
+        },
+        {
+            "id": "action2",
+            "name": "Action [Error]",
+            "description": "Type: Action\n\nDescription: If something goes wrong, this fires",
+            "types": ["action"]
+        },
+        {
+            "id": "message",
+            "name": "Message",
+            "description": "Type: Object, List\n\nDescription: The message obtained. If \"Split Message\" is enabled, this will return a list containing all message objects instead of a single one.",
+            "types": ["object", "list"]
+        }
+    ],
+
+    code(cache) {
+        const channel = this.GetInputValue("channel", cache);
+        const text = this.GetOptionValue("text", cache) + "";
+        const text1 = this.GetInputValue("text1", cache) + "";
+        const text2 = this.GetInputValue("text2", cache) + "";
+        const text3 = this.GetInputValue("text3", cache) + "";
+        const text4 = this.GetInputValue("text4", cache) + ""; 
+        const text5 = this.GetInputValue("text5", cache) + "";
+        const text6 = this.GetInputValue("text6", cache) + "";
+        const text7 = this.GetInputValue("text7", cache) + "";
+        const text8 = this.GetInputValue("text8", cache) + ""; 
+        const text9 = this.GetInputValue("text9", cache) + "";
+        const text10 = this.GetInputValue("text10", cache) + "";
+        const text11 = this.GetInputValue("text11", cache) + "";
+        const text12 = this.GetInputValue("text12", cache) + ""; 
+        const text13 = this.GetInputValue("text13", cache) + "";
+        const text14 = this.GetInputValue("text14", cache) + "";
+        const text15 = this.GetInputValue("text15", cache) + "";
+        const text16 = this.GetInputValue("text16", cache) + ""; 
+        const text17 = this.GetInputValue("text17", cache) + "";
+        const text18 = this.GetInputValue("text18", cache) + "";
+        const text19 = this.GetInputValue("text19", cache) + "";
+        const text20 = this.GetInputValue("text20", cache) + ""; 
+        let chars = {'${text1}':text1,'${text2}':text2,'${text3}':text3,'${text4}':text4,'${text5}':text5,'${text6}':text6,'${text7}':text7,'${text8}':text8,'${text9}':text9,'${text10}':text10,'${text11}':text11,'${text12}':text12,'${text13}':text13,'${text14}':text14,'${text15}':text15,'${text16}':text16,'${text17}':text7,'${text18}':text18,'${text19}':text19,'${text20}0':text20};
+        let s = text;
+        s = s.replace (/(\${text1})|(\${text2})|(\${text3})|(\${text4})|(\${text5})|(\${text6})|(\${text7})|(\${text8})|(\${text9})|(\${text10})|(\${text11})|(\${text12})|(\${text13})|(\${text14})|(\${text15})|(\${text16})|(\${text17})|(\${text18})|(\${text19})|(\${text20})/g, m => chars[m]);
```

Comment:
what about 
```let s = eval("`" + text.replace(/`/g, "\\`") + "`")``` ?
thats why we implememnt it with `${var}` and the abillity to get keys is also given